frontend/src/components/layout/Header.tsx

```diff
@@ -248,19 +259,27 @@ const Header = () => {
                       <div>
                         <Menu.Button className="flex rounded-full bg-white">
                           <span className="sr-only">{t("open-user-menu")}</span>
-                          <Image
-                            className="rounded-full"
-                            src={
-                              Object.values(user.info.auths).find(
-                                (auth) => auth.avatar,
-                              ).avatar
-                            }
-                            width="38"
-                            height="38"
-                            alt={t("user-avatar", {
-                              user: user.info.displayname,
-                            })}
-                          />
+                          {userAvatarUrl ? (
```

Comment:
This part should probably be moved to a new component, so that we don't need to repeat it (for e.g. in the other file) and keep it in sync